Calibration/IsolatedParticles/test/proto_runIsolatedParticlesGeneratedJets_cfg.py

```diff
@@ -22,16 +22,18 @@
 
 ##################### digi-2-raw plus L1 emulation #########################
 process.load("Configuration.StandardSequences.Services_cff")
-process.load('Configuration/StandardSequences/GeometryExtended_cff')
-process.load('Configuration/StandardSequences/MagneticField_38T_cff')
-process.load('TrackingTools/TrackAssociator/DetIdAssociatorESProducer_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_cff')
+process.load('TrackingTools.TrackAssociator.DetIdAssociatorESProducer_cff')
 
 #################### Conditions and L1 menu ################################
 
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['run2_mc']
```

Comment:
You are using a Run2 simulation GT with Run1 simulated events (very old, BTW...)